arangod/Graph/GraphOperations.cpp

```diff
@@ -302,13 +298,19 @@ OperationResult GraphOperations::addOrphanCollection(VPackSlice document, bool w
                    std::string{TRI_errno_string(TRI_ERROR_GRAPH_COLLECTION_USED_IN_EDGE_DEF)}));
   }
 
+  // add orphan collection to graph
+  _graph.addOrphanCollection(std::string(collectionName));
```

Comment:
```suggestion
  _graph.addOrphanCollection(collectionName);
```

Reply:
@jsteemann does not compile without the extra string copy, because the function requests a move. And afterwards, the value is still used.

Reply:
Additionally, line 342 (trx.finish(TRI_ERROR_NO_ERROR);, not part of this PR) looks downright wrong and should be removed entirely.

What happens if trx.begin fails? What should be done otherwise?

Reply:
just removed that line I mentioned.
transactions will abort automatically if not committed and the trx object goes out of scope.